snprc_ehr/resources/queries/study/SPF_Results.query.xml

```diff
@@ -0,0 +1,22 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="SPF_Results" tableDbType="TABLE">
+                <tableTitle>SPF Results</tableTitle>
+                <columns>
+                    <column columnName="Date">
+                        <columnTitle>Bleed Date</columnTitle>
+                        <formatString>Date</formatString>
+                    </column>
+                    <column columnName="Id">
```

Comment:
Hi Terry. Can i please know if this ColumnName 'Id' can convey more details instead of just Id or do we need to have the same name as it is a foreign key from study.animal Table - Id? Can we add a <ColumnTitle> attribute to this Id ?

Reply:
previously i gave the columnTitle in open and closed tags (<>) and it is not visible in those tags.

Reply:
Id is the name used in LabKey for the animal id. We want to keep that the same.